32-fall-detector-and-panic-button/README.md

```diff
@@ -1,89 +1,86 @@
 
-![breadboard button wiring](assets/breadboard_button.jpg "Breadboard Button Wiring")
+![breadboard button wiring](images/breadboard_button.jpg "Breadboard Button Wiring")
 
-Now, when the button is pressed, it will connect A2 to GND.
+Now, when the button is pressed, it will connect F_D12 to GND.
 
 The accelerometer board has a hole on it labeled INT1. This is the interrupt line that our app will use to detect a fall event, so we need to connect it to the reference node. To do that:
 
 1. Take a male-to-male jumper wire that fits the hole. We've found that a [standard DuPont jumper wire](https://www.amazon.com/SUNKEE-Dupont-Color-Jumper-2-54mm/dp/B00AX3PHV6) with a square shape may not fit this hole. If that's the case, you can try using a [jumper wire with a rounded shape](https://www.amazon.com/Breadboard-Jumper-Wire-75pcs-pack/dp/B0040DEI9M).
 2. Push one end of the wire as far as it'll go through the hole. This is so that when it's soldered in, we can plug it into the breadboard.
 3. Using your soldering iron and flux, solder the jumper into place.
 4. Seat the accelerometer onto the breadboard using the exposed end of the wire sticking out under the accelerometer board.
-5. Connect the other end of the wire to pin A1 on the reference node.
+5. Connect the other end of the wire to pin F_D13 on the Notecarrier-F.
 
-The accelerometer also has a Qwiic port that needs to be wired up to the Sparrow reference node. Plug the Qwiic breadboard jumper cable into this port, and then on the other end make the following connections:
+The accelerometer also has a Qwiic port that needs to be wired up to the Swan Plug the Qwiic jumper cable into this port, and then on the other end to the Qwiic port on the Swan.
 
-* Connect the black wire to GND via the breadboard's minus rail.
-* Connect the red wire to VIO on the node.
-* Connect the blue wire to SDA on the node.
-* Connect the yellow wire to SCL on the node.
+Now, everything is wired up. It should look something like this, although I soldered a header onto my BMA400 board, rather than soldering the wire directly:
 
-Now, everything is wired up. It should look something like this:
+![breadboard full wiring](images/breadboard_full.jpg "Breadboard Full Wiring")
 
-![breadboard full wiring](assets/breadboard_full.jpg "Breadboard Full Wiring")
+Since this is a fall detector, we're going to need to drop this assembly from a decent height to simulate a person falling. To keep things secure, we secure everything to the breadboard with rubber bands or tape.
```

Comment:
```suggestion
Since this is a fall detector, we're going to need to drop this assembly from a decent height to simulate a person falling. To keep things from bouncing around, you can secure everything to the breadboard with rubber bands or tape.
```